bazel/protobuf.bzl

```diff
@@ -28,10 +28,8 @@ def _protoc_cc_output_files(proto_file_sources):
     for p in proto_file_sources:
         basename = p.basename[:-len(".proto")]
 
-        cc_hdrs.append(basename + ".pb.h")
         cc_hdrs.append(basename + ".pb.validate.h")
 
-        cc_srcs.append(basename + ".pb.cc")
```

Comment:
Change looks good, but a bit confused, since I'd expect that the validation code needs to have the headers from the protoc stubs to be able to access values..

Reply:
It does, the validation code declares a dependency on the cc_proto_library rule. This line is just because the invocation of `protoc` isn't expected to generate these files.